Sources/PackageManager/FtpConnection.php

```diff
@@ -293,7 +293,7 @@ public function passive(): bool
 		}
 
 		// This is pretty simple - store it for later use ;).
-		$this->pasv = ['ip' => $match[1] . '.' . $match[2] . '.' . $match[3] . '.' . $match[4], 'port' => $match[5] * 256 + $match[6]];
+		$this->pasv = ['ip' => $match[1] . '.' . $match[2] . '.' . $match[3] . '.' . $match[4], 'port' => (int) ($match[5] * 256 + $match[6])];
```

Comment:
Casting the result of the expression to int is unnecessary. The result of the arithmetic operation will already be an integer.